### ISSUE TITLE
As a User...

### ISSUE THREAD

**Issue by nbolser** [nbolser/note#1]
- [ ] 1. I would like to have the ability to use OAuth2 to log in. 
- [ ] 2. I would like to CRUD a note from a UI
- [ ] 3. I would like to CRUD a note via API
- [ ] 4. I would like to export notes to JSON

**Comment by woodall** [comment]
I started thinking about the API for **0.0.1** 
How would you authenticate?

I'm thinking we should consider that as it's own feature, maybe as **0.0.2**?
(Other than _format_ as json)

**Comment by nbolser** [comment]
Yes, agreed. 0.0.1 would be MVP (auth, app) and then 0.0.2 would be API with auth from 0.0.1.
